src/pages/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import "./NotFound.scss";
+
+function NotFoundPage() {
+  return (
+    <div className="error-page">
+      <div className="error-content">
+        <h1 className="error-code">404</h1>
```

Comment:
every h1, p and button tag should be replaced with the common component you've created